packages/kbn-optimizer/src/__fixtures__/mock_repo/plugins/bar/public/legacy/styles.scss

```diff
@@ -0,0 +1,4 @@
+body {
+  size: $globalStyleConstant;
```

Comment:
Are you also testing for invalid CSS markup? Just checking if it matters that `size` is not real property.

Reply:
Haha, good catch, it didn't really matter because node-sass compiled it just fine but I've updated it to `width:`